apps/api/v1/pages/api/bookings/_get.ts

```diff
@@ -177,9 +189,15 @@ function buildWhereClause(
     };
   }
 
-  return {
-    ...whereClause,
-  };
+  // Add upcoming filter if requested
```

Comment:
There is no need to add `// Add upcoming filter if requested` comment when code explains it already - when reading `if (upcomingOnly) {` it is already clear what will happen : )